app/src/main/res/layout/fragment_drawer.xml

```diff
@@ -0,0 +1,38 @@
+<com.google.android.material.navigation.NavigationView
+  android:id="@+id/nav_view"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:layout_gravity="start"
+  android:fitsSystemWindows="true"
+  app:headerLayout="@layout/nav_header_navigation_drawer"
+  app:itemBackground="@android:color/transparent"
+  app:itemIconTint="@color/drawer_item"
+  app:itemTextColor="@color/drawer_item"
+  app:menu="@menu/navigation_drawer_menu"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_gravity="bottom"
+    android:orientation="horizontal"
+    android:padding="16dp">
+
+    <ImageView
+      android:layout_width="@dimen/image_dim_24"
+      android:layout_height="@dimen/image_dim_24"
+      android:layout_gravity="center_vertical"
+      android:layout_margin="@dimen/margin_10"
+      android:src="@drawable/ic_settings_grey_48dp" />
+
+    <TextView
+      android:id="@+id/logout"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_gravity="center_vertical"
+      android:layout_marginLeft="@dimen/margin_30"
+      android:text="@string/admin_settings"
+      android:textColor="@color/black" />
+  </LinearLayout>
+</com.google.android.material.navigation.NavigationView>
```

Comment:
Add one extra line just before this line

Reply:
resolved